src/core/frames/frame_controller.ts

```diff
@@ -122,6 +122,10 @@ export class FrameController implements AppearanceObserverDelegate, FetchRequest
   // Link interceptor delegate
 
   shouldInterceptLinkClick(element: Element, url: string) {
+    const linkMethod = element.getAttribute("data-turbo-method") || element.getAttribute("data-method")
```

Comment:
This should probably be extracted as it is also used [here](https://github.com/hotwired/turbo/pull/299/files#diff-4fdcafb26950e01d47ac3215a38de2bbb3d4788ca980214293f334cfec78a7c0R140)

Where would be the best place to do so ?

Reply:
I've actually wanted to drop compatibility with data-method, as it actually makes coexisting with UJS more confusing. So we should just proceed with data-turbo-method as the only attribute, and then we don't need an extraction. Then we can even inline it:

```javascript
if (element.getAttribute("data-turbo-method")) {
  return false
} else {
  return this.shouldInterceptNavigation(element)
}
```

Then we also prevent multiple exit wounds.